tools/fregrid/fregrid.c

```diff
@@ -73,20 +73,19 @@ char *usage[] = {
   "          [--associated_file_dir dir] [--format format]                               ",
   "          [--deflation #] [--shuffle 1|0]                                             ",
   "                                                                                      ",
-  "fregrid remaps data (scalar or vector) from input_mosaic onto                         ",
-  "output_mosaic.  Note that the target grid also could be specified                     ",
-  "through lonBegin, lonEnd, latBegin, latEnd, nlon and nlat. Currently                  ",
-  "only T-cell scalar regridding and AGRID vector regridding is                          ",
-  "available.  Bilinear interpolation is implemented only for cubic grid                 ",
-  "vector interpolation. The interpolation algorithm used is controlled                  ",
-  "by --interp_method with default 'conserve_order1'. Currently the                      ",
-  "'conserve_order1', 'conserve_order2' and 'bilinear' remapping schemes                 ",
-  "are implemented. 'bilinear' is only used to remap data from cubic grid                ",
-  "to latlon grid. Alternative schemes can be added if needed. fregrid                   ",
-  "expects NetCDF format input. scalar_field and/or u_field/v_field must                 ",
-  "be specified. u_fld and v_fld must be paired together.                                ",
-  "If using NetCDF4, output will have the same deflation and shuffle settings            ",
-  "unless specified.                                                                     ",
+  "fregrid remaps data (scalar or vector) from an input_mosaic onto an output_mosaic,    ",
+  "but the output (or target) grid can also be specified through the inputs              ",
+  "lonBegin, lonEnd, latBegin, latEnd, nlon and nlat. Currently only T-cell scalar       ",
+  "regridding and AGRID vector regridding is available. Vector fields can be mapped      ",
```

Comment:
This was something that was a little unclear to me in reading the documentation before, so I think this may be worth being explicit about too:
```suggestion
  "regridding and AGRID vector regridding (where the vector field is expressed in spherical coordinates) is available. Vector fields can be mapped      ",
```

Reply:
Thanks. Will use suggestion.